packages/opal-common/opal_common/schemas/data.py

```diff
@@ -9,6 +10,11 @@
 JsonableValue = Union[Dict[str, Any], List[Any]]
 
 
+class HttpMethodsAllowed(Enum):
```

Comment:
needs to include `str` as a base class as well,  otherwise pydantic fails to serialize this to json sometimes.

Reply:
do you are more information about this "fail sometimes"?
I check the code anyway and if i am right, we never serialize this class to json (or class that use it).
There is no point to have an Enum if we include str.

Reply:
we do serialize this to json within permit (which is not open source) - in general we should always strive to make pydantic models be able to serialize correctly.